src/Apps/Artwork/Components/__stories__/OtherWorks.story.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { Section } from "Styleguide/Utils/Section"
+import { OtherWorksQueryRenderer as OtherWorks } from "../OtherWorks"
+import { OtherWorksByArtistQueryRenderer as OtherWorksByArtist } from "../OtherWorks/OtherWorksByArtist"
+// import { OtherWorksFromAuctionQueryRenderer as OtherWorksFromAuction } from "../OtherWorks/OtherWorksFromAuction"
+// import { OtherWorksFromFairQueryRenderer as OtherWorksFromFair } from "../OtherWorks/OtherWorksFromFair"
+// import { OtherWorksFromGalleryQueryRenderer as OtherWorksFromGallery } from "../OtherWorks/OtherWorksFromGallery"
+// import { RelatedWorksQueryRenderer as RelatedWorks } from "../OtherWorks/RelatedWorks"
+
+storiesOf("Styleguide/Artwork/OtherWorks", module)
+  .add("<OtherWorks />", () => {
+    return (
+      <>
+        <Section title="OtherWorks (Auction Artwork)">
+          <OtherWorks artworkID="achille-perilli-phantom" />
```

Comment:
Going back to my comments above, here we would add `isAuctionArtwork={true}`, or whatever, as well.